main.go

```diff
@@ -33,10 +39,86 @@ func main() {
 	sync := marathon.NewMarathonSync(remote, consul)
 	go sync.Sync()
 
-	// set up routes
+	fh := &ForwardHandler{consul}
+
+	version, err := remote.Version()
+	if version >= "0.9.0" {
```

Comment:
since this is a lexical comparison, it'll fail for "0.10.0", correct? How could we fix this for future versions?